administrator/components/com_admin/script.php

```diff
@@ -6435,6 +6435,10 @@ public function deleteUnexistingFiles($dryRun = false, $suppressOutput = false)
             '/plugins/system/webauthn/webauthn.php',
             '/plugins/task/checkfiles/checkfiles.php',
             '/plugins/task/demotasks/demotasks.php',
+            // From 4.2.0 to 4.2.1
+            '/media/vendor/hotkeys.js/js/hotkeys.js',
+            '/media/vendor/hotkeys.js/js/hotkeys.min.js',
```

Comment:
```suggestion
            '/media/vendor/hotkeys.js/js/hotkeys.min.js',
            '/media/vendor/hotkeys.js/js/hotkeys.min.js.gz'
```
The pre-zipped file should be removed, too.

That's why I always base my deleted files and folders checks on comparing the full installation zip packages of the 3 releases, and that's the only it can be done, because in the build/tmp folder, the build.php script deletes files from the build between building the full installation packages and the update packages, so checking the folder with the build after that will not cover all files.